english/explore-outside/talk-child/talk-child.md

```diff
@@ -7,3 +7,5 @@ Do you:
 
 [Take your cellphone out, call Tank and ask for 
 the nearest exit out of the Matrix.](../journey-to-flaming-tower/exit/exit.md)
+
+[Try a MacGyver escape move with items in pocket.](../macgyver/macgyver.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.